extensions/amp-form/0.1/amp-form.js

```diff
@@ -192,7 +197,10 @@ function installSubmissionHandlers(win) {
 function installAmpForm(win) {
   return getService(win, 'amp-form', () => {
     if (isExperimentOn(win, TAG)) {
-      installStyles(win.document, CSS, () => installSubmissionHandlers(win));
+      installStyles(win.document, CSS, () => {
+        installSubmissionHandlers(win);
+        actionServiceFor(win).addEvent('submit');
```

Comment:
From what I can tell - multiple actions are supported, separated by `;`. I filed the #3783 to confirm. Could you please confirm and update the docs?